connman/src/inet.c

```diff
@@ -2223,6 +2223,16 @@ int __connman_inet_ipv6_do_dad(int index, int timeout_ms,
 	return err;
 }
 
+int connman_inet_ipv6_do_dad(int index, int timeout_ms,
+				struct in6_addr *addr,
+				connman_inet_ns_cb_t callback,
+				void *user_data)
+{
+	return __connman_inet_ipv6_do_dad(index, timeout_ms, addr,
+					(__connman_inet_ns_cb_t)callback,
```

Comment:
Not really. I just wasn't sure if we'd need this permanently as it might be necessary to have CLAT at some point as a internal component instead of plugin. To avoid this, needed to make 35772b0cb6f90ce9df71f70caffd46aa3007f5ab which removes the need.